### PR TITLE
update commands,js added tipbot commands

### PR DESCRIPTION
**Pull request by ykris45**
please review this PR and if any mistake of coding , tell me , based on mr thomas idea in #help. thanks Gbu

### CODE REVIEW

**Comment by ykris45** [comment]
please ignore this commit, there is a small error 
https://github.com/lbryio/lbry-wunderbot/pull/112/commits/45d5a55d3f77a3e860c3032cbbe2a7a398610005

**Comment by ykris45** [comment]
#111

**Comment by tzarebczan** [comment]
Thanks Chris!

@filipnyquist can you make sure the syntax is correct :p

**Comment by ykris45** [comment]
you welcome sir. i hope the syntax corrected

**Comment by MSFTserver** [comment]
if your gonna use `bot-sandboxChannel` you need create helper and set it in the botfile

**Comment by ykris45** [comment]
@MSFTserver 
https://github.com/lbryio/lbry-wunderbot/pull/112/commits/3b91da102cb43386e6099d5e9eb197c919093196 thanks for say "perfect"

**Comment by ykris45** [comment]
Refereced pr i was deleted and make one bundle with this PR .

File: bot/modules/commands.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ let config = require("config");
 let miningChannel = config.get("Channels").mining;
 let randomChannel = config.get("Channels").random;
 let verificationChannel = config.get("Channels").verification;
+let bot-sandboxChannel = config.get("Channels").sandbox;
```
Comment (filipnyquist) [inline bot/modules/commands.js]: please keep the styling a bit and change to for example botSandboxChannel

**Comment by ykris45** [comment]
let bot-sandboxChannel = config.get("Channels").botsandbox;
Like this filip or
let bot-sandboxChannel = config.get("Channels").bot-sandbox;

**Comment by tzarebczan** [comment]
@ykris45 I'll take care of it, thanks

**Comment by ykris45** [comment]
okay my leader @tzarebczan . collabs again

File: bot/modules/commands.js (hunk 1 above)
Comment (tzarebczan) [reply bot/modules/commands.js]: fixed

**Comment by tzarebczan** [comment]
Should be all set @filipnyquist

**Comment by MSFTserver** [comment]
since i dont have access these lines should be changed to since this is already opened, its for welcome.js 
image
should be changed to 
```
if (!msg.mentions.members.first()) {
  msg.channel.send("no user defined");
  return;
}
```

**Comment by nikooo777** [comment]
all the changes in commands.js are superseded by this PR https://github.com/lbryio/lbry-wunderbot/pull/116

I'd suggest removing that file before merging

**Comment by tzarebczan** [comment]
I'm just gonna close this PR and fix the minor things in a new one.

**Pull request closed without merging**